src/widgets/Finance/components/Finance.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const Finance = ({ finance }) =>
+  <div className="panel Number">
+    <div className="panel-heading">Rocket Internet SE Stock Price</div>
```

Comment:
Please add the company name as an optional config param as its not available in the Google Finance payload.